pages/vi/profiles/Maniblitz.md

```diff
@@ -1,8 +1,8 @@
-# Hey, I am Karl Kevin Tiba Fossoh :us:
-![img](https://www.jwhomes.com/~/media/JW%20Homes/Georgia/Atlanta/Marietta/The%20Quad%20at%20Meeting%20Park/Carousel/Quad_at_Meeting_Park_by_John_Wieland_Marietta_Square.ashx)
-#### :clock5: Marietta, Georgia, U.S.A. | Time Zone: Eastern Daylight Time(EDT)
-###### :computer: Using macOS High Sierra version 10.13.3
-> Yes I am a developer, not an artist
+# Hey, I am Karl Kevin Tiba Fossoh
+![The beautiful city of Marietta, with his timeless park](https://www.jwhomes.com/~/media/JW%20Homes/Georgia/Atlanta/Marietta/The%20Quad%20at%20Meeting%20Park/Carousel/Quad_at_Meeting_Park_by_John_Wieland_Marietta_Square.ashx)
+#### Marietta, Georgia, U.S.A. | Time Zone: Eastern Daylight Time(EDT)
+###### Using macOS High Sierra version 10.13.3
+ > Yes I am a developer, not an artist
```

Comment:
Whitespace is normally referred as a blank line. Hence Add blank Line before and after this so that the quote will render.
Also remove the unnecessary whitespace on line 28.